### PR TITLE
Enhance/refactor errorMsg

### PR DESCRIPTION
**Pull request by DanSallau**
Hi,

On my attempt to refactor our errorMsg component and to avoid inline operations . I made few changes to the files below. Should there be any suggestion with regard to this please let me know would be glad to address.

### CODE REVIEW

**Comment by borisyankov** [comment]
Using state for this is definitely not correct.

File: src/_common/ErrorMsg.js [hunk 1]
```diff
@@ -7,9 +7,9 @@ export default class ErrorMsg extends Component {
 		shown: PropTypes.bool.isRequired,
 		text: PropTypes.string.isRequired,
 	};
-
 	render() {
 		const { shown, text } = this.props;
-		return shown ? <p className="errorfield"><M m={text} /></p> : <span />;
+        const strText = text.split(')').length > 1 ? text.split(')')[1] : text;
+		return shown ? <p className="errorfield"><M m={strText} /></p> : <span />;
```
Comment (borisyankov) [inline src/_common/ErrorMsg.js]: if not shown return null, and not <span />

File: src/_common/ErrorMsg.js [hunk 2]
```diff
@@ -7,9 +7,9 @@ export default class ErrorMsg extends Component {
 		shown: PropTypes.bool.isRequired,
 		text: PropTypes.string.isRequired,
 	};
-
 	render() {
 		const { shown, text } = this.props;
-		return shown ? <p className="errorfield"><M m={text} /></p> : <span />;
+        const strText = text.split(')').length > 1 ? text.split(')')[1] : text;
```
Comment (borisyankov) [inline src/_common/ErrorMsg.js]: Extract into a function in the _utils folder. Like errorToString and put this there, plust a couple of unit tests :)

**Comment by borisyankov** [comment]
Two comments, rest is fine now :)

File: src/_common/ErrorMsg.js [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 
 	static propTypes = {
 		shown: PropTypes.bool.isRequired,
 		text: PropTypes.string.isRequired,
 	};
-
 	render() {
 		const { shown, text } = this.props;
-		return shown ? <p className="errorfield"><M m={text} /></p> : <span />;
+		return shown ? <p className="errorfield"><M m={errorToString(text)} /></p> : null;
```
Comment (borisyankov) [inline src/_common/ErrorMsg.js]: We don't need to translate errors. 
They are already translated on the server ;)

Comment (nuruddeensalihu) [inline src/_common/ErrorMsg.js]: Some are not server errors. Just hard coded

**Pull request merged**